ci/deps/travis-37-cov.yaml

```diff
@@ -15,7 +14,6 @@ dependencies:
   # pandas dependencies
   - beautifulsoup4
   - botocore>=1.11
-  - cython>=0.29.16
```

Comment:
didn't we already update this?

Reply:
> didn't we already update this?

Yes, we did. Cython version was sepecifed twice in this file. I didn't see this line when I bumped cython in #36296.